assets/terraform/azure/bootstrap.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+#
+# File passed to VM at creation time
+#
+apt update
+apt install -y python-pip lvm2
+pip install --upgrade awscli
+
+set -euo pipefail
+
+pvcreate -ff /dev/sdc
+vgcreate data /dev/sdc
+lvcreate -n var_lib_gravity -l 20%VG data
+lvcreate -n var_lib_data -l 20%VG data
+lvcreate -n etcd -l 30%VG data
```

Comment:
same issue with etcd -- I know disks on azure are pretty different (worse), so maybe doesn't apply, but I would assume it does.